tests/test_utils.py

```diff
@@ -450,3 +450,14 @@ def tmp_function():
     # If XTB is not in $PATH then the function should not execute
     assert len(test_list) == 0
     os.environ["PATH"] = path_env_var
+
+    # if XTB is in $PATH, the function should execute
+    xtbpath = shutil.which("xtb")
+    if xtbpath is None:
+        return
+
+    assert xtbpath.lower().endswith(("xtb", "xtb.exe"))
+
+    tmp_function()
+
+    assert len(test_list) == 1
```

Comment:
I'm going to contradict myself and suggest for the sake of code coverage this is less flat than it could be.. how do you feel about it?

```suggestion
    if xtbpath is not None:
         assert xtbpath.lower().endswith(("xtb", "xtb.exe"))
         tmp_function()
         assert len(test_list) == 1
```

Reply:
Yes, that works too.